src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/InitiatePaymentTest.java

```diff
@@ -0,0 +1,162 @@
+package uk.gov.hmcts.cmc.claimstore.controllers;
+
+import feign.FeignException;
+import org.junit.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.ResultActions;
+import uk.gov.hmcts.cmc.claimstore.BaseSaveTest;
+import uk.gov.hmcts.cmc.domain.models.ClaimData;
+import uk.gov.hmcts.cmc.domain.models.ioc.InitiatePaymentResponse;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimData;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.cmc.ccd.domain.CaseEvent.INITIATE_CLAIM_PAYMENT_CITIZEN;
+import static uk.gov.hmcts.cmc.claimstore.utils.ResourceLoader.successfulCoreCaseDataStoreStartResponse;
+import static uk.gov.hmcts.cmc.claimstore.utils.ResourceLoader.successfulCoreCaseDataStoreSubmitResponse;
+
+@TestPropertySource(
+    properties = {
+        "document_management.url=false",
+        "feature_toggles.ccd_async_enabled=false",
+        "feature_toggles.ccd_enabled=true",
+        "feature_toggles.async_event_operations_enabled=false"
```

Comment:
As async operation is in prod now and we are not going to turn that off. We should add tests with this flag set true as default. No need to turn that off for new tests.

Reply:
ah right - I think I copy pasted that so there might be another offending test. I'll take a look

Reply:
will set it to true 👍